stories/Button.stories.jsx

```diff
@@ -12,12 +12,13 @@ export default {
 export const Variants = () => {
   const variants = [
     'primary',
-    'outline',
-    'outlineAlt',
-    'positive',
+    'large',
     'link',
     'linkSecondary',
-    'linkPositive'
+    'linkPositive',
+    'outline',
+    'outlineAlt',
+    'positive'
```

Comment:
I cannot find anywhere in the new styleguide the outline and outline alt variants. We should map these values (for backwards compatibility) with the `primary` style.

Reply:
You mean to map all these: 
    'linkSecondary',
    'linkPositive',
    'outline',
    'outlineAlt',
    'positive'
with the primary style?